korman/properties/modifiers/region.py

```diff
@@ -203,6 +203,237 @@ def requires_actor(self):
+                          options=set())
+
+    environment_size = FloatProperty(name="Environment Size", description="Environment Size",
+                                     default=7.5, min=1.0, max=100.0)
+    environment_diffusion = FloatProperty(name="Environment Diffusion", description="Environment Diffusion",
+                                          default=1.0, min=0.0, max=1.0)
+    room = IntProperty(name="Room", description="Room",
+                       default=-1000, min=-10000, max=0)
+    room_hf = IntProperty(name="Room HF", description="Room High Frequency",
+                          default=-100, min=-10000, max=0)
+    room_lf = IntProperty(name="Room LF", description="Room Low Frequency",
+                          default=0, min=-10000, max=0)
+    decay_time = FloatProperty(name="Decay Time", description="Decay Time",
+                               default=1.49, min=0.1, max=20.0)
+    decay_hf_ratio = FloatProperty(name="Decay HF Ratio", description="Decay High Frequency Ratio",
+                                   default=0.83, min=0.1, max=2.0)
+    decay_lf_ratio = FloatProperty(name="Decay LF Ratio", description="Decay Low Frequency Ratio",
+                                   default=1.0, min=0.1, max=2.0)
+    reflections = IntProperty(name="Reflections", description="Reflections",
+                              default=-2602, min=-10000, max=1000)
+    reflections_delay = FloatProperty(name="Reflections Delay", description="Reflections Delay",
+                                      default=0.007, min=0.0, max=0.3)
+    reverb = IntProperty(name="Reverb", description="Reverb",
+                         default=200, min=-10000, max=2000)
+    reverb_delay = FloatProperty(name="Reverb Delay", description="Reverb Delay",
+                                 default=0.011, min=0.0, max=0.3)
+    echo_time = FloatProperty(name="Echo Time", description="Echo Time",
+                              default=0.25, min=0.1, max=0.5)
+    echo_depth = FloatProperty(name="Echo Depth", description="Echo Depth",
+                               default=0.0, min=0.0, max=1.0)
+    modulation_time = FloatProperty(name="Modulation Time", description="Modulation Time",
+                                    default=0.25, min=0.1, max=5.0)
+    modulation_depth = FloatProperty(name="Modulation Depth", description="Modulation Depth",
+                                     default=0.0, min=0.0, max=1.0)
+    air_absorption_hf = FloatProperty(name="Air Absorption HF", description="Air Absorption High Frequency",
+                                      default=-5.0, min=-10.0, max=0.0)
+    hf_reference = FloatProperty(name="HF reference", description="High Frequency Reference",
+                                 default=5000.0, min=1000.0, max=20000.0)
+    lf_reference = FloatProperty(name="LF reference", description="Low Frequency Reference",
+                                 default=250.0, min=20.0, max=1000.0)
```

Comment:
These all need an `options=set()` to prevent folks from adding spurious keyframes.